kolibri/plugins/setup_wizard/assets/src/machines/wizardMachine.js

```diff
@@ -475,97 +462,67 @@ export const wizardMachine = createMachine(
+      addImportedUser: assign((ctx, event) => ({
+        importedUsers: uniq([...ctx.importedUsers, event.value]),
+      })),
+      setFirstLodUser: assign((_, event) => ({
+        firstImportedLodUser: event.value,
+      })),
+      setLodAdmin: assign((_, event) => ({
+        lodAdmin: {
+          username: event.value.adminUsername,
+          password: event.value.adminPassword,
+          id: event.value.id,
         },
-      }),
-      setLodSuperAdmin: assign({
+      })),
+      setLodSuperAdmin: assign((ctx, event) => ({
         // Sets the super admin to be set as the device super admin -- the first LOD user imported
-        superuser: (ctx, event) => {
-          if (!ctx.superuser) {
-            return {
-              username: event.value.username,
-              password: event.value.password,
-            };
-          } else {
-            return ctx.superuser;
-          }
+        superuser: ctx.superuser || {
+          username: event.value.username,
+          password: event.value.password,
         },
-      }),
-      setRemoteUsers: assign({
-        remoteUsers: (_, event) => event.value.users,
-      }),
+      })),
+      setRemoteUsers: assign((_, event) => ({
+        remoteUsers: event.value.users,
+      })),
       /**
        * Assigns the machine to have the initial context again while maintaining the value of
        * canGetOsUser.
 
        * This effectively resets the machine's state
        */
-      resetContext: assign(initialContext),
+      resetContext: assign(() => initialContext),
       setImportedFacility: assign({
-        isImportedFacility: () => {
-          return true;
-        },
+        isImportedFacility: () => true,
       }),
     },
     guards: {
       // Functions used to return a true/false value. When the functions are called, they are passed
       // the current value of the machine's context as the only parameter
-      isOnMyOwnOrGroup: context => {
-        return context.onMyOwnOrGroup === Presets.PERSONAL;
-      },
-      isGroupSetup: context => {
-        return context.onMyOwnOrGroup === UsePresets.GROUP;
-      },
+      isOnMyOwnOrGroup: context => context.onMyOwnOrGroup === Presets.PERSONAL,
+      isGroupSetup: context => context.onMyOwnOrGroup === UsePresets.GROUP,
       canGetOsUser: () => checkCapability('get_os_user'),
-      isNewFacility: context => {
-        return context.facilityNewOrImport === FacilityTypePresets.NEW;
-      },
-      isImportFacility: context => {
-        return context.facilityNewOrImport === FacilityTypePresets.IMPORT;
-      },
-      isLodSetup: context => {
-        return context.fullOrLOD === DeviceTypePresets.LOD;
-      },
-      isFullSetup: context => {
-        return context.fullOrLOD === DeviceTypePresets.FULL;
-      },
+      isNewFacility: context => context.facilityNewOrImport === FacilityTypePresets.NEW,
+      isImportFacility: context => context.facilityNewOrImport === FacilityTypePresets.IMPORT,
```

Comment:
I notice that the function passed to `guard` seems to be expecting an object on the first param ([relevant docs](https://stately.ai/docs/guards#inline-guards)) shaped like `{ context, event }` -- but here we only handle one param as if it's the context.

Makes me wonder if there might be issues uncovered by tests in the setup wizard because of this.